app/assets/javascripts/controllers/dialog_user/dialog_user_controller.js

```diff
@@ -1,30 +1,47 @@
   var vm = this;
 
   vm.$onInit = function() {
-    var apiCall = new Promise(function(resolve) {
-      var url = '/api/service_dialogs/' + dialogId +
-        '?resource_action_id=' + resourceActionId +
-        '&target_id=' + targetId +
-        '&target_type=' + targetType;
-
-      resolve(API.get(url, {expand: 'resources', attributes: 'content'}).then(init));
-    });
-
-    Promise.resolve(apiCall).then(miqService.refreshSelectpicker);
+    var url = '/api/service_dialogs/' + dialogId +
+      '?resource_action_id=' + resourceActionId +
+      '&target_id=' + targetId +
+      '&target_type=' + targetType;
+
+    API.get(url, {expand: 'resources', attributes: 'content'})
+      .then(function (data) { vm._dialog = data.content[0] })
+      .then(loadServiceRequest)
+      .then(postprocess)
+      .then(function () {
+        vm.dialog = vm._dialog;
+        delete vm._dialog;
+        vm.dialogLoaded = true;
+      });
   };
 
-  function init(dialog) {
-    vm.dialog = dialog.content[0];
-    vm.dialogLoaded = true;
+  function postprocess(replace) {
+    var promises = [];
 
-    _.forEach(vm.dialog.dialog_tabs, function(tab) {
+    _.forEach(vm._dialog.dialog_tabs, function(tab) {
       _.forEach(tab.dialog_groups, function(group) {
         _.forEach(group.dialog_fields, function(field) {
-          const replaceField = dialogReplaceData ? JSON.parse(dialogReplaceData).find(function (replace) { return replace.name === field.name }) : false;
-          if (replaceField) {
-            field.default_value = replaceField.value;
+          // If the service request has been copied, replace the fields' default values from the original service request
+          if (dialogReplaceRequestId) {
+            var replaceField = replace.find(function (item) { return item.name === field.name });
+            if (replaceField) {
+              field.default_value = replaceField.value;
+            }
           }
+
+          // Load tag values from the API
+          if (field.type == 'DialogFieldTagControl') {
+            promises.push(API.get('/api/categories/' + field.options.category_id + '/tags?expand=resources&attributes=categorization').then(function(response) {
+              field.values = response.resources.map(function(tag) {
+                return { value: tag.id, name: tag.name, description: tag.categorization.description };
```

Comment:
`                return { id: tag.id, name: tag.name, description: tag.categorization.description };` should work

dialog-user is using `fieldValue.id as fieldValue.description for fieldValue in vm.dialogField.values` so `value` is ignored